src/components/IstioConfigCard/IstioConfigCard.tsx

```diff
@@ -64,30 +64,32 @@ class IstioConfigCard extends React.Component<Props> {
       return this.noIstioConfig();
     }
     let rows: IRow[] = [];
-    this.props.items.map((item, itemIdx) => {
-      rows.push({
-        cells: [
-          {
-            title: (
-              <span>
-                <PFBadge badge={IstioTypes[item.type].badge} position={TooltipPosition.top} />
-                {this.overviewLink(item)}
-              </span>
-            )
-          },
-          {
-            title: (
-              <ValidationObjectSummary
-                id={itemIdx + '-config-validation'}
-                validations={item.validation ? [item.validation] : []}
-                style={{ verticalAlign: '-0.5em' }}
-              />
-            )
-          }
-        ]
+    this.props.items
+      .sort((a: IstioConfigItem, b: IstioConfigItem) => (a.type < b.type ? -1 : a.name < b.name ? -1 : 1))
```

Comment:
Not sure if this is correct, so name should fall only when == 0, otherwise you may have some random sort.

Reply:
In this case if type of a is lower show a first. Otherwise compare name ....this is wrong

I think that this should be:
If the type is the same then compare name, if the name is the same too then not care about order, a and then b or b and then a